### ISSUE TITLE
Certain sizing should be based on REM instead of PX for accessibility

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-css#1201]
## Description
Update type relative sizing to be delivered in REM instead of PX.

## Why do you need this feature or component?
To better support the accessible delivery of applications leveraging Spectrum CSS for both components and layout tokens.

**Comment by castastrophe** [comment]
@pfulton Would this likely require an update to the page package as well to define a base font size of which our values could be relative? And maybe adding a new variable: `--spectrum-global-dimension-static-font-size-root` mapping to the root px value then updating the static dimensions to rems.

```css
body {
    font-size: var(--spectrum-global-dimension-static-font-size-root);
    /* equivalent to --spectrum-global-dimension-static-font-size-75, aka 12px at medium scale, 15px at large */
}

.spectrum {
  --spectrum-global-dimension-static-font-size-50: 0.92rem;
  --spectrum-global-dimension-static-font-size-75: 1rem;
  --spectrum-global-dimension-static-font-size-100: 1.17rem;
    /* etc... */
}
```

[Responsive typography](https://www.element84.com/blog/responsive-typography#:~:text=and%20rem.-,Best%20of%20both%20worlds,-The%20best%20parts)

**Comment by castastrophe** [comment]
This is an update that needs to be driven by design. They are aware of the request and conversations are ongoing but this is not planned work at this time.
